solutions/2.5-huffman-solutions.ipynb

```diff
@@ -0,0 +1,45 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### _Programming Exercise 2.5.3:_ Compute height of TapLeafs in TapTree"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tapscripts = [(\"A\", tapleafA.script), (\"B\", tapleafB.script), (\"C\", tapleafC.script), (\"D\", tapleafD.script), (\"E\", tapleafE.script)]\n",
+    "for n, script in tapscripts:\n",
+    "    controlblockA = control_map[script]\n",
```

Comment:
I think just controlblock is ok since we are iterating and folks might get confused since we have an A TapLeaf